qiskit_optimization/applications/graph_optimization_application.py

```diff
@@ -94,9 +95,9 @@ def graph(self) -> nx.Graph:
         return self._graph
 
     @staticmethod
+    @deprecate_method("0.3.0", DeprecatedType.FUNCTION)
```

Comment:
Version 0.3.0 is correct. As soon as the release goes out we update the main branch version file so aside from a very short period at release time, it should be always be the next version https://github.com/Qiskit/qiskit-optimization/blob/207d3215cca04f8006d65644a644c85adc9611c7/qiskit_optimization/VERSION.txt#L1

Can we include that they should just use `nx.gnm_random_graph` directly as a replacement

Reply:
Oh, I didn't know we have such a file. Thank you!
Also, I updated to use `nx.gnm_random_graph` directly.

Reply:
> Can we include that they should just use nx.gnm_random_graph directly as a replacement

Sorry I did not mean a code change to directly return the value, I was thinking that in the deprecation we should tell the user that they can use nx.gnm_random_graph as a replacement (i.e. they can directly call that method rather than using this, now deprecated, method)

Reply:
I see xD I added additional message to guide users to use NetworkX directly. It's now like this.
```
DeprecationWarning: The random_graph method is deprecated as of version 0.3.0 and will be removed no sooner than 3 months after the release. Instead use the netowrkx.gnm_random_graph function in NetworkX, directly.
```